frontend/src/Pages/Inductees.svelte

```diff
@@ -191,6 +190,16 @@ let years = [
                     </select>
```

Comment:
Best not to hard code >2027 option. Is there a way to not hardcode? Or even remove this one entirely since the fields are auto populated?